lib/install/config/webpack/development.js

```diff
@@ -1,9 +1,21 @@
 // Note: You must restart bin/webpack-dev-server for changes to take effect
 
 const merge = require('webpack-merge')
+const { readFileSync } = require('fs')
 const sharedConfig = require('./shared.js')
 const { settings, output } = require('./configuration.js')
 
+const { dev_server: devServer } = settings
+let https = devServer.https
+
+// Set ssl key and certificates if supplied
+if (devServer.ssl_key_path && devServer.ssl_cert_path) {
+  https = {
+    key: readFileSync(devServer.ssl_key_path),
+    cert: readFileSync(devServer.ssl_cert_path)
+  }
+}
```

Comment:
@dhh There were some issues filed where people are using HTTPS for development, mainly for cases where they are interacting with API's that is hosted on HTTPS